xc-admin/packages/xc-admin-common/src/__tests__/MultisigInstruction.test.ts

```diff
@@ -0,0 +1,416 @@
+import { ChainName } from "@certusone/wormhole-sdk";
```

Comment:
For more clarity I suggest having separate test files per class

Reply:
Ok